.gitlab/e2e.yml

```diff
@@ -262,6 +262,14 @@ new-e2e-cws-dev:
   # Temporary, until we manage to stabilize those tests.
   allow_failure: true
 
+new-e2e-cws-main:
+  extends: .new_e2e_template
+  rules: !reference [.on_main_and_no_skip_e2e]
+  variables:
```

Comment:
You should also set the `TEAM` variable.
It is used to tag AWS resources and track cost.

Reply:
Ah right, should be good in the last commit